src/drt/src/ta/FlexTA_init.cpp

```diff
@@ -740,45 +740,6 @@ void FlexTAWorker::initFixedObjs()
       } else if (type == frcBlockage || type == frcInstBlockage) {
         bloatDist = initFixedObjs_calcBloatDist(obj, layerNum, bounds);
         initFixedObjs_helper(box, bloatDist, layerNum, nullptr);
-
-        if (DBPROCESSNODE == "GF14_13M_3Mx_2Cx_4Kx_2Hx_2Gx_LB") {
```

Comment:
Why do you think this code is unused?

Reply:
It's not effective. Each ta worker has either vertical tracks only or horizontal tracks only. This part adds costs for the upper and lower metal layer of the current layer (which is gonna be in the opposite direction) so it will never find any tracks to add cost to.

Reply:
Also I ran a secure CI to make sure that removing this part won't change a thing.